src/components/ButtonPanel/ButtonPanel.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Button from '../Button/Button'
+import './ButtonPanel.css';
+
+const ButtonPanel = (props) => {
```

Comment:
Nice job here! At the risk of sounding like a broken record, let's use object destructuring for props.